action.yml

```diff
@@ -37,43 +41,50 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - name: Get token
+      id: codeartifact-token
+      shell: bash
+      run: |
+        if [ "${{ inputs.codeartifact-dummy-token }}" == "false" ]; then
+          AWS_TOKEN=$(aws codeartifact get-authorization-token --domain ${{ inputs.codeartifact-domain }} --domain-owner ${{ inputs.codeartifact-domain-owner }} --region ${{ inputs.aws-region }} --query authorizationToken --output text)
+          echo "token=$AWS_TOKEN" >> $GITHUB_OUTPUT
+        else
+          echo "We use fake token for testing purpose only!!!" >> $GITHUB_STEP_SUMMARY
+          echo "token=${{ inputs.codeartifact-dummy-token }}" >> $GITHUB_OUTPUT
+        fi
 
     - name: Fetch CodeArtifact metadata
       id: codeartifact-metadata
       shell: bash
       run: |
-        AWS_TOKEN=$(aws codeartifact get-authorization-token --domain ${{ inputs.codeartifact-domain }} --domain-owner ${{ inputs.codeartifact-domain-owner }} --region ${{ inputs.aws-region }} --query authorizationToken --output text)
         REPO_URI="${{ inputs.codeartifact-domain }}-${{ inputs.codeartifact-domain-owner }}.d.codeartifact.${{ inputs.aws-region }}.amazonaws.com/pypi/${{ inputs.codeartifact-repository }}/"
         REPO_URL="https://$REPO_URI"
-        echo "token=$AWS_TOKEN" >> $GITHUB_OUTPUT
         echo "repo-uri=$REPO_URI" >> $GITHUB_OUTPUT
         echo "repo-url=$REPO_URL" >> $GITHUB_OUTPUT
 
     - name: Configure PIP
       shell: bash
       run: |
-        pip config set site.index-url "https://aws:${{ steps.codeartifact-metadata.outputs.token }}@${{ steps.codeartifact-metadata.outputs.repo-uri }}simple/"
+        pip config set site.index-url "https://aws:${{ steps.codeartifact-token.outputs.token }}@${{ steps.codeartifact-metadata.outputs.repo-uri }}simple/"
```

Comment:
step is named codeartifact-metadata - not codeartifact-token

Reply:
I split step for testing purpose
Llook here:
https://github.com/vimn-public/aws-codeartifact-login-action/pull/4/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R44-R45